extensions/hardening/k3s/harden_nodes.go

```diff
@@ -40,14 +37,12 @@ func HardenNodes(nodes []*nodes.Node, nodeRoles []string, kubeVersion string) er
 		}
 
 		if strings.Contains(nodeRoles[key], "--controlplane") {
-			logrus.Infof("Copying over files to node %s", node.NodeID)
-			user, err := user.Current()
-			if err != nil {
-				return nil
-			}
-
-			dirPath := filepath.Join(user.HomeDir, "go/src/github.com/rancher/shepherd/extensions/hardening/k3s")
-			err = node.SCPFileToNode(dirPath+"/audit.yaml", "/home/"+node.SSHUser+"/audit.yaml")
+			_, err = node.ExecuteCommand(`sudo bash -c 'cat << EOF > /home/` + node.SSHUser + `/audit.yaml
```

Comment:
I think this yaml should live in an actual file that's then read into this call instead of raw string.

Reply:
Jenkins specifically was calling this an issue and couldn't find the file. Removing it was the only way it passed successfully. Locally, it works. through Jenkins, it doesn't.

Reply:
Talked offline on this point. This is most likely being moved to actions once that becomes a thing. When that happens, I'll convert this back into files. Jenkins won't have this issue once that happens.